src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -1624,8 +1661,13 @@ static int xgq_freq_scaling_by_topo(struct platform_device *pdev,
 	    ARRAY_SIZE(target_freqs), target_freqs[0], target_freqs[1],
 	    target_freqs[2], target_freqs[3]);
 
-	return xgq_freq_scaling(pdev, target_freqs, ARRAY_SIZE(target_freqs),
-		verify);
+	ret = set_and_verify_freqs(pdev, target_freqs, ARRAY_SIZE(target_freqs),verify);
```

Comment:
This function name doesn't make sense. If we look at the original code. for example:
https://github.com/Xilinx/XRT/blob/master/src/runtime_src/core/pcie/driver/linux/xocl/subdev/clock_wiz.c#L1005
The name should be clock_scaling. Recommend to not change this code. Remain unchanged.